SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -273,16 +273,24 @@ FLocalDeploymentManager::ERuntimeStartResponse FLocalDeploymentManager::StartLoc
 	// Use the runtime start timestamp as the log directory, e.g. `<Project>/spatial/localdeployment/<timestamp>/`
 	FString LocalDeploymentLogsDir = FPaths::Combine(SpatialGDKServicesConstants::LocalDeploymentLogsDir, RuntimeStartTime.ToString());
 
+	// Store these logs alongside the GDK ones for convenience
+	const TCHAR* RuntimeEventLogPaths = TEXT("EventTracing/runtime");
+	FString EventTracingPath = FPaths::ConvertRelativePathToFull(FPaths::Combine(FPaths::ProjectSavedDir(), RuntimeEventLogPaths));
+	IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
+	if (!PlatformFile.CreateDirectoryTree(*EventTracingPath))
+	{
+		UE_LOG(LogSpatialDeploymentManager, Log, TEXT("Failed to create runtime event log path."));
```

Comment:
Is this really a log category log? not a warning or an error?

Reply:
Upgraded to error, should always be writable!